### DISCUSSION TITLE
Hide tabs (and fields) from Live Preview

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9164] [Ideas]
> Created by: Ian (i.young@me.com) on 2015/08/20 08:36:31 +0000
> Votes at time of UserVoice import: 7

-----------------------------------

Live Preview mode includes fields from **all** tabs. Is it possible to specify tabs (and/or) to be left out of the Live Preview display?

For example, if you have an 'SEO' tab which consists of meta data it's not relevant to include this in the Live Preview mode whilst constructing the page. With lots of tabs this just becomes confusing for the user when only a few of them are relevant.

Originally asked here: http://craftcms.stackexchange.com/questions/11062/live-preview-hide-tabs-from-live-preview?noredirect=1#comment14392_11062

**Comment by angrybrad** [comment]
Related: https://github.com/craftcms/cms/issues/948

**Comment by schwarzdesign** [comment]
@brandonkelly Why was this closed? Is this issue fixed / obsolete with Craft 3.7, or was the requested feature turned down? Closing an issue without any comment regarding its status results in a dead end for people looking for the same problem – like me right now :)

**Comment by brandonkelly** [comment]
@schwarzdesign Sorry about that. The issue was closed due to an automated script that was supposed to convert it to a [Discussion](https://github.com/craftcms/cms/discussions/categories/ideas). For whatever reason a few issues were only closed but not converted, this one included apparently.

I’ve reported it to GitHub. They’ve been very responsive so this should be sorted soon.

**Comment by schwarzdesign** [comment]
@brandonkelly Thanks for the reply! So should I open a new discussion for this? Tab titles in preview editors would be really nice to have. Currently the field layout gets confusing when fields that are normally in their own tabs are just displayed below the main context without the tab title being visible.

**Comment by brandonkelly** [comment]
No need to open a discussion.
